.circleci/config.yml

```diff
@@ -0,0 +1,26 @@
+version: 2.1
+orbs:
+  docker: circleci/docker@0.5.19
+  helm-tools: pennlabs/helm-tools@0.1.8
+
+branch-filters: &branch-filters
+  filters:
+    branches:
+      only:
+        - master
+
+workflows:
+  version: 2
+  build-and-deploy:
+    jobs:
+      - docker/publish:
```

Comment:
@KeenWill the common funding application has test cases, you should run these before deploying. See penn-clubs for how Django test cases are set up.